data/sources/00_areatree

```diff
@@ -557,11 +578,18 @@
       6101:Heizkraftwerk:
       6102:Betriebsgebäude:
       6103:Hochspannungsgebäude:
+      6104:Feuerwehr TUM Garching (Hallen West):
       6106:Trafostation Betriebsgebäude (TBG):
       6107:Garagenbau:
-    6202:Feuerwehrwache:
+    62:Feuerwehrwache:feuerwehrwache[joined_building]
```

Comment:
I am unshure if `feuerwehrwache` is a too general id

Reply:
Yes maybe `feuerwehrwache-garching` is better? Both 61.. and 62.. belong to the the Werksfeuerwehr. I'm not sure how exactly they are separated.